app/services/childCRUD.ts

```diff
@@ -173,14 +174,15 @@ export const setActiveChild = async (languageCode: any, uuid: any, dispatch: any
         child.parent_gender = userParentalRole[0].value
       }
       const allDatatoStore = await getAllDataToStore(languageCode, dispatch, "AddEditChild", child);
-      console.log("allDatatoStore AddEditChild1--", allDatatoStore);
+      console.log("allDatatoStore AddEditChild1--", child);
```

Comment:
we expect this console.log(); to be deleted in the production,

Reply:
okay sure